docusaurus.config.js

```diff
@@ -407,6 +407,10 @@ const config = {
 						from: '/en/connect-a-ui/metabase-and-clickhouse',
 						to: '/en/integrations/metabase',
 					},
+					{
+						from: '/en/connect-a-ui/explo-and-clickhouse',
```

Comment:
I don't think we need this redirect for new pages

Reply:
Done!